3MI0800303/practicum-1.cpp

```diff
@@ -0,0 +1,148 @@
+#include <iostream>
+#pragma warning(disable : 4996)
+
+struct OrderedTwo {
+    int firstNumber;
+    int secondNumber;
+    bool isBigger(OrderedTwo& ordered) const;
+};
+
+inline bool OrderedTwo::isBigger(OrderedTwo& cmp) const {
+    if (firstNumber == cmp.firstNumber) return secondNumber > cmp.secondNumber;
+    if (firstNumber < cmp.firstNumber) return false;
+    return true;
+}
+
+int getBiggest(OrderedTwo ords[100]) {
+    for (size_t i = 0; i < 100; i++) {
```

Comment:
Typically when you have something that goes with the name `find...` or `get` you don't really expect it to have a side effect. This is not the case here - your original array `ords` may get re-ordered as a result of the use of `swap`. At the same time the name is quite misleading - you return a fixed value (0) that is of type `int` whereas the name suggests you'll be returning an instance of an `OrderedTwo` . Not that I am particularly good with naming things but I'd say this one is better named `findIndexOfBiggestElement` (ofc. you need to lose the side effect - changing the order of the elements in the array).